app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,7 @@
     <color name="colorPrimary">#424242</color>
     <color name="colorPrimaryDark">#1B1B1B</color>
     <color name="colorAccent">@color/colorPrimary</color>
+    <color name="color_8D6E63">#3E2C26</color>
```

Comment:
Why did you choose this peculiar name styling for the color resources?

Reply:
@arctouch-matheusfaleiro my mistake
Not the right way